src/Project/ToolRunner.ts

```diff
@@ -161,7 +161,6 @@ export class ToolRunner {
         // TODO(jyoung): Need password encryption
         tool = `echo ${process.env.userp} | sudo -S ` + tool;
         this.child = cp.spawn(tool, toolargs, {cwd: path, shell: true});
-        process.env.userp = '';
```

Comment:
If there is already installed package, it should be removed first.
In this case, this PR requests the two requests using one job to be entered the password from the user only once. However, this code resets the user password before all requests are complete. So I remove this line and reset the password after all requests are done. (JobRunner.ts L102)